handler_error.go

```diff
@@ -87,6 +87,12 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 
 		t := time.Now()
 
+		if ctxGetDefaultVersion(r) {
+
+			vinfo := ctxGetVersionInfo(r).Name
```

Comment:
might worth to check if `ctxGetVersionInfo(r)` returns nil, looking at implementation it can. I see from code that assumption is that we always set VersionInfo in context, if we ever miss it - we hit into nil pointer panic